packages/notification-services-controller/src/NotificationServicesPushController/services/services.ts

```diff
@@ -1,13 +1,11 @@
 import log from 'loglevel';
 
-import type { Types } from '../../NotificationServicesController';
 import type { PushNotificationEnv } from '../types';
+import type {
+  CreateRegToken,
+  DeleteRegToken,
+} from '../types/push-service-interface';
 import * as endpoints from './endpoints';
-import type { CreateRegToken, DeleteRegToken } from './push';
-import {
-  listenToPushNotificationsClicked,
-  listenToPushNotificationsReceived,
-} from './push/push-web';
```

Comment:
In the initial design, we did a decent start at decoupling the webby parts from this controller (to make it agnostic). However the issue was we were directly importing and using the web module directly!